README.md

```diff
@@ -7,27 +7,34 @@ GopherJS - A compiler from Go to JavaScript
 
 GopherJS compiles Go code ([golang.org](https://golang.org/)) to pure JavaScript code. Its main purpose is to give you the opportunity to write front-end code in Go which will still run in all browsers.
 
+### What's new?
+
+ - 2021-04-04: **Go 1.16 is now officially supported!** 🎉 🎉 🎉
+
 ### Playground
+
 Give GopherJS a try on the [GopherJS Playground](http://gopherjs.github.io/playground/).
 
 ### What is supported?
-Nearly everything, including Goroutines ([compatibility table](https://github.com/gopherjs/gopherjs/blob/master/doc/packages.md)). Performance is quite good in most cases, see [HTML5 game engine benchmark](https://ajhager.github.io/engi/demos/botmark.html). Cgo is not supported.
+
+Nearly everything, including Goroutines ([compatibility documentation](https://github.com/gopherjs/gopherjs/blob/master/doc/compatibility.md)). Performance is quite good in most cases, see [HTML5 game engine benchmark](https://ajhager.github.io/engi/demos/botmark.html). Cgo is not supported.
 
 ### Installation and Usage
-GopherJS requires Go 1.12 or newer.
+
+GopherJS requires Go 1.16 or newer.
```

Comment:
Do we want to add a sentence about using an old version if you require something older?  Maybe with a link to the GitHub tags or releases page?

Reply:
Added a link to the explanation in docs/compatibility.md and a lint to GitHub Releases.